src/controllers/users_controller.js

```diff
@@ -5,23 +5,29 @@ module.exports.setUp = () => {
 };
 
 module.exports.findAllUsers = (req, res) => {
-    res.json(usersModel.findAllUsers());
+    usersModel.findAllUsers((err, users) => {
+        res.status(err ? 500 : 200).json(users);
+    });
 };
 
 module.exports.register = ({body: {name = null, email = null, password = null}}, res) => {
-    res.json(usersModel.registerUser(name, email, password));
+    if (!email || !password || !name) {
+        return res.status(400).json("Email, name and Password must be provided");
+    }
+    usersModel.registerUser(name, email, password, (err) => {
+        res.status(err ? 500 : 200).json();
```

Comment:
Isn't it supposed to return an object here?

Reply:
status should suffice